Code-of-Conduct.md

```diff
@@ -5,6 +5,13 @@
 
 # Warum haben wir einen Code of Conduct
 
+Die K4CG hat es sich zum Ziel gemacht, einen diskriminierungsfreien 
+Raum für alle Menschen zu schaffen. Wir tolerieren in keiner Weise 
+die Belästigung oder Diskriminierung. 
```

Comment:
Wuerde entweder "...in keiner Weise Belaestigung..." ohne "die" schreiben oder evtl. noch ein "die Belaestigung oder Diskriminierung ...von..." ergaenzen

Reply:
+1 fuer "ohne die"

Reply:
Ich habs entfernt